ChangeLog

```diff
@@ -15,7 +15,17 @@ Release date: TBA
 What's New in Pylint 2.4.3?
 ===========================
 
-Release date: TBA
+* Allow parallel linting when run under Prospector
```

Comment:
Can we move this to the 2.5 section instead?

Reply:
done